persist-cli/src/main/java/io/ballerina/persist/PersistToolsConstants.java

```diff
@@ -41,8 +41,13 @@ private PersistToolsConstants() {
     public static final String KEY_DATABASE = "database";
     public static final String KEY_HOST = "host";
 
+    public static final String HOST_PLACEHOLDER = "${%s.clients.host}";
+    public static final String PORT_PLACEHOLDER = "${%s.clients.port}";
+    public static final String PASSWORD_PLACEHOLDER = "${%s.clients.password}";
+    public static final String USER_PLACEHOLDER = "${%s.clients.user}";
+    public static final String DATABASE_PLACEHOLDER = "${%s.clients.database}";
+
```

Comment:
Shall we categorise the constants into required inner classes? I believe having these inside one Placeholder class is more clearer. Lets do it as part of the restructure @sahanHe

Reply:
sure let's proceed with the restructure